configure.ac

```diff
@@ -238,6 +238,39 @@ AX_VALGRIND_DFLT([sgcheck], [off])
 # Add a target to run testsuite with valgrind.
 AX_VALGRIND_CHECK()
 
+# Requirements for running test-crypt-kat-gen.py.
+ENABLE_CRYPT_KAT_GEN="no"
+PYTHON_BIN="python3 python"
+
+for ver in `seq 0 1 10`; do
+  PYTHON_BIN="python3.$ver $PYTHON_BIN"
+done
+
+AC_PATH_PROGS([PYTHON], [$PYTHON_BIN], [])
+
+if test x"$PYTHON" != "x"; then
+  AX_PROG_PYTHON_VERSION([3.6], [ENABLE_CRYPT_KAT_GEN="yes"],
+    [AC_MSG_WARN([Python >= 3.6 is required for optional Makefile targets.])
+  ])
+
+  if test x"$ENABLE_CRYPT_KAT_GEN" = "xyes"; then
+    for pymod in array ctypes hashlib multiprocessing os passlib re sys; do
```

Comment:
It should only be necessary to do this test for `passlib`.  All the other modules are part of the Python standard library.